app/views/unattended/report_templates/host_-_compare_content_hosts_packages.erb

```diff
@@ -15,41 +15,48 @@ template_inputs:
   value_type: plain
   resource_type: Katello::ActivationKey
 model: ReportTemplate
+require:
+- plugin: katello
+  version: 3.16.1
```

Comment:
does it really require 3.16.1? it seems to be working with 3.16.0 too, I'm not sure if 3.16.1 is even available already - https://fedorapeople.org/groups/katello/releases/yum/3.16/katello/el7/x86_64/

Reply:
I think that `sortable_version` will only be available in 3.16.1 - https://projects.theforeman.org/issues/30561